src/api/app/services/worker_status_service/jobs_statistics_fetcher.rb

```diff
@@ -0,0 +1,33 @@
+module WorkerStatusService
+  class JobsStatisticsFetcher
+    attr_reader :ws
+
+    class << self
+      def call
+        new.statistics
+      end
+    end
+
+    def initialize
+      @ws = Hash.from_xml(worker_status)
```

Comment:
https://apidock.com/rails/v6.0.0/Hash/from_xml/class maybe better than the Xmlhash gem?

Reply:
Nokogiri might also be an option here.

Reply:
Nokogiri is an option, however, to use Nokogiri, to transform the data manually to hash is, IMO, more error prone than using a Rails core method to do that

Reply:
> https://apidock.com/rails/v6.0.0/Hash/from_xml/class maybe better than the Xmlhash gem?

define 'better' here. XmlMini is a pure ruby implementation and unless you plan to add the 3rd libxml using gem, it's for sure super slow: https://www.rubydoc.info/docs/rails/ActiveSupport/XmlMini

So what bothers you with xmlhash that you feel the need to pull the handbrake?

Reply:
Let's pull in the `libxml-ruby` gem then. I prefer to rely on a gem which is actively maintained and widely used.

We want to eventually remove the `xmlhash` gem anyway.

Reply:
> unless you plan to add the 3rd libxml using gem, it's for sure super slow: https://www.rubydoc.info/docs/rails/ActiveSupport/XmlMini

You mean, maintained 3rd library https://github.com/xml4r/libxml-ruby vs unmaintained 3rd library https://github.com/coolo/xmlhash? We all know the answer. But this PR isn't about it, but about sending backend metrics to our monitoring system. Do you want to help me to review this PR @coolo ? :)

Reply:
Before anyone exchanges or introduces anything, one should know what impact it has on performance right? :)